Packs/CommonScripts/Scripts/FormatURL/README.md

```diff
@@ -1,4 +1,5 @@
 Strips, unquotes and unescapes URLs. If the URL is a Proofpoint or ATP URL, extracts its redirect URL.
+If more than one URL is passed to the formatter the separator must be a pipe ("|")
```

Comment:
```suggestion
If more than one URL is passed to the formatter, the separator must be a pipe ("|").
```
let's add this to the script "comment" field as well

Reply:
Also added to the comment.